config/bootstrap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Cake\Event\EventManager;
+
+EventManager::instance()->on(
+    'Dispatcher.beforeDispatch',
+    ['priority' => 51],
+    function ($event) {
+        $controller = false;
+        if (isset($event->data['controller'])) {
+            $controller = $event->data['controller'];
+        }
+        if ($controller) {
+            $callback = ['Muffin\Webservice\Model\EndpointRegistry', 'get'];
+            $controller->modelFactory('Endpoint', $callback);
+        }
+    }
+);
```

Comment:
This came from https://github.com/cakephp/elastic-search/blob/master/config/bootstrap.php . I'll write a DispatchFilter for this